README.md

```diff
@@ -1,6 +1,14 @@
-# walledeval
+# Walled Eval: Testing LLMs Against Jailbreaks and Unprecedented Harms
 
-> _Test LLMs against jailbreaks and unprecedented harms_
+<p align="center">
+  <img width="700" alt="walleai_logo_shield" src="https://github.com/walledai/walledeval/assets/32847115/d8b1d14f-7071-448b-8997-2eeba4c2c8f6">
```

Comment:
This is way too big, also should either be above the title or at the bottom, not here.